CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.9.2
+
+- Change grafana-plugin-sdk-go version
```

Comment:
Should we maybe say revert rather than change and explain what bug this will fix and any bugs we know it brings back?